mpt_trie/src/debug_tools/stats.rs

```diff
@@ -323,7 +323,7 @@ mod tests {
 
     #[test]
     fn hand_made_trie_has_correct_node_stats() {
```

Comment:
nit: can have the test return `TrieOpResult` and just use `?` instead of `assert!(_.is_ok())` and `unwrap()`.